common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -65,7 +65,9 @@ public final class NativeLibraryLoader {
         String workdir = SystemPropertyUtil.get("io.netty.native.workdir");
         if (workdir != null) {
             File f = new File(workdir);
-            f.mkdirs();
+            if (!f.exists() && !f.mkdirs()) {
+             throw new ExceptionInInitializerError(new IOException("Custom native workdir mkdirs failed: " + workdir));
```

Comment:
Indentation looks off

Reply:
indeed. Fixed.
By the way, have you ever considered using something like spotless to automatically format from maven?